helpers/client_list.go

```diff
@@ -12,16 +13,25 @@ import (
 
 var (
 	nodeCache = make(map[string]*api.Node)
+	stderrLog *log.Logger
 )
 
+func init() {
+	// We make an logger that _always_ print to stderr to ensure CLI calls like
+	// "nomad-helper stats --output-format json | jq '.'" works. All progress and filtering
+	// output will go to stderr now instead of stdout for debug/info etc
+	stderrLog = log.New()
+	stderrLog.Out = os.Stderr
+}
+
 func FilteredClientList(client *api.Client, c *cli.Context) ([]*api.NodeListStub, error) {
-	log.Info("Finding legible nodes")
+	stderrLog.Info("Finding legible nodes")
```

Comment:
now that you are at this, can you change `legible` (readable) to `eligible` (can be chosen) please?

Reply:
Can't find what you can't read though ;)